master/pom.xml

```diff
@@ -1813,13 +1815,43 @@
         <groupId>org.sakaiproject.basiclti</groupId>
         <artifactId>basiclti-util</artifactId>
         <version>${sakai.version}</version>
-        <!-- soon to be provided -->
+        <scope>provided</scope>
       </dependency>
       <dependency>
         <groupId>org.sakaiproject.basiclti</groupId>
         <artifactId>basiclti-common</artifactId>
         <version>${sakai.version}</version>
-        <!-- soon to be provided -->
+        <!-- Depends on ResourceLoader so must be in each webapp -->
+      </dependency>
+      <dependency>
+        <groupId>com.nimbusds</groupId>
+        <artifactId>nimbus-jose-jwt</artifactId>
+        <version>${sakai.nimbus.jose.jwt.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.jsonwebtoken</groupId>
+        <artifactId>jjwt-api</artifactId>
+        <version>${sakai.io.jsonwebtoken.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.jsonwebtoken</groupId>
+        <artifactId>jjwt-impl</artifactId>
+        <version>${sakai.io.jsonwebtoken.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.jsonwebtoken</groupId>
+        <artifactId>jjwt-jackson</artifactId>
+        <version>${sakai.io.jsonwebtoken.version}</version>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
isn't some of these deps already in master?

Reply:
Yup - You can't be too careful :) Fixed.